app/home/home.component.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {provide} from 'angular2/core';
+import {it, describe, expect, injectAsync, beforeEachProviders} from 'angular2/testing';
+import {TestComponentBuilder, ComponentFixture, setBaseTestProviders} from 'angular2/testing';
```

Comment:
two imports from the same place, to make the lines shorter?
Is there a shortcut that is just all of them, like `ANGULAR_TESTING_ALL_THE_THINGS`?